pkgs/top-level/all-packages.nix

```
@@ -16940,4 +16940,8 @@ with pkgs;
   rustdesk-flutter = callPackage ../by-name/ru/rustdesk-flutter/package.nix {
     flutter = flutter324;
   };
+
+  davis = callPackage ../by-name/da/davis/package.nix {
```

```suggestion
  davis = callPackage ../by-name/da/davis/package.nix {
```
Please re-check alphabetical order.

---

Nevermind! `all-packages.nix` has no alphabetical order atm.